README.md

```diff
@@ -74,37 +49,57 @@ Run these commands:
 
 `sudo apt-get purge amdgpu-install`
 
-## Step 10:
+## Step 7:
 Download the latest drivers.
 
 Run this command:
 
-`wget http://repo.radeon.com/amdgpu-install/5.6/ubuntu/jammy/amdgpu-install_5.7.50700-1_all.deb`
+`wget https://repo.radeon.com/amdgpu-install/23.20.00.48/ubuntu/jammy/amdgpu-install_5.7.00.48.50700-1_all.deb`
 
-## Step 11:
+## Step 8:
 Now install AMDGPU.
 
 Run these commands:
 
-`sudo dpkg -i amdgpu-install_5.7.50700-1_all.deb`
+`sudo apt install ./amdgpu-install_5.7.00.48.50700-1_all.deb`
 
-`amdgpu-install --usecase=hip,rocm  --no-32`
+`amdgpu-install -y --usecase=graphics,rocm`
```

Comment:
Is `hip` not wanted?

Reply:
We can add it but not needed for sd at least. Its like that in their documentation I provided in the pr.

Reply:
How about for stuff like LLM training or using something like the `bitsandbytes-rocm` fork which I think is hipified.

Reply:
Then add it, better smth to much than to little.